### PR TITLE
🧰 Update `GetSuggestedRoutes` call

### PR DESCRIPTION
**Pull request by shivekkhurana**
This is to reflect changes made in https://github.com/status-im/status-go/pull/5002/files

### Summary
In the implementation of swaps, @saledjenic had to update the signature of the get suggested routes function. 
This PR implements that change on status-mobile side. 

The change request was made here: https://github.com/status-im/status-go/pull/5002#issuecomment-2042144504

### Review notes
This is not a major change. Just a sync to stay compatible with new status-go. 

### Testing notes
This change will effect the sending of tokens, specially across chains so that should be tested. We need to ensure that we get some route suggestions without any errors.

#### Areas that maybe impacted
- Multi chain transfers
- Same chain transfers

##### Functional
- wallet / transactions

status: ready

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

File: src/status_im/contexts/wallet/send/events.cljs [hunk 1]
```diff
@@ -168,6 +169,7 @@
                                   to-address
                                   amount-in
                                   token-id
+                                  to-token-id
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/send/events.cljs:180]: should there not be a corresponding Status-Go update with this?

Comment (shivekkhurana) [reply src/status_im/contexts/wallet/send/events.cljs:180]: There is, it's alteady merged. I mentioned it in the description.

Comment (J-Son89) [reply src/status_im/contexts/wallet/send/events.cljs:180]: Yes but we should bump the version of status go in this pr too, no?

**Comment by briansztamfater** [comment]
I think we should request manual qa, wdyt?

**Comment by J-Son89** [comment]
> I think we should request manual qa, wdyt?

It would also be good if QA can prioritise this to avoid bringing this issue sneaking into develop

File: src/status_im/contexts/wallet/send/events.cljs (hunk 1 above)
Comment (shivekkhurana) [reply src/status_im/contexts/wallet/send/events.cljs:180]: Oh, sorry. I assumed mobile pulls latest status-go. On it.

**Comment by shivekkhurana** [comment]
> > I think we should request manual qa, wdyt?
> 
> It would also be good if QA can prioritise this to avoid bringing this issue sneaking into develop

I've marked for manual QA. 

🛎️  pinging @churik to prioritise this to prevent a leaky cascade.

**Review by J-Son89 — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
Hi @shivekkhurana thank you for PR. No issues from my side. Ready to be merged

**Comment by shivekkhurana** [comment]
Merging this after resolving conflicts.

**Pull request merged**